Update/_mina_001.txt

```diff
@@ -1074,17 +1047,16 @@ void main()
 		   NULL, " Keiichi experienced it...", Line_Normal);
 	ClearMessage();
 	DisableWindow();
-	ChangeBustshot( 2, "Kei_Def1#", 200, TRUE );
-	DisplayWindow();
+	ChangeBustshot( 2, "sprite/normal/kei2_def1_", 200, TRUE );
 
 	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#6972c1>梨花</color>", NULL, "<color=#6972c1>Rika</color>", NULL, Line_ContinueAfterTyping); }
 	PlayVoice(4, "ps3/s12/05/180500024", 128);
 	OutputLine(NULL, "「…………………？」",
 		   NULL, "\".........?\"", Line_Normal);
 	ClearMessage();
 
-	DrawBustshotWithFiltering( 2, "Sa_AkireruB1#", "maskright", 1, 160, 0, FALSE, 0, 0, 0, 0, 0, 10, 0, FALSE );
-	DrawSceneWithMask( "Mo2", "maskright", 1, 0, 300 );
+	DrawBustshot( 0, "sprite/normal/sa2b_akireru_b1_", 160, 0, 0, FALSE, 0, 0, 0, 0, 0, 0, 0, 5, 0, FALSE );
+	DrawSceneWithMask( "background/Mo2", "maskleft", 1, 0, 300 );
```

Comment:
use lowercase path

```suggestion
	DrawSceneWithMask( "background/mo2", "maskleft", 1, 0, 300 );
```

Reply:
was unsure, changed all to lowercase now 👌